oak-auth-external/src/main/java/org/apache/jackrabbit/oak/spi/security/authentication/external/basic/DefaultSyncContext.java

```diff
@@ -522,7 +522,7 @@ protected void syncMembership(@NotNull ExternalIdentity external, @NotNull Autho
         while (grpIter.hasNext()) {
             Group grp = grpIter.next();
             if (isSameIDP(grp)) {
-                declaredExternalGroups.put(grp.getID(), grp);
+                declaredExternalGroups.put(grp.getID().toLowerCase(), grp);
```

Comment:
Done. Whoever has this issue will be spammed with warnings, but I guess they have to live with that.

Reply:
hi @mbaedke , i am not sure i can follow you on this one..... what is the new group-ids list for? and why would someone get spammed with warnings? sorry.... i don't get it.

what i had in mind is something along the following lines:
```
Group previous = declaredExternalGroups.put(grp.getID().toLowerCase(), grp);
if (previous != null) {
     log.warn("Replacing group '{}' with same case-insensitive name '{}'", previous.getId(), grp.getID());
}

```

Reply:
Hi @anchela, I don't understand. Your code would do nothing, because declaredExternalGroups is empty initially.
Re spamming: the warnings will stay, because the ids do not change in the repo or in the LDAP.